doc/iterations/2024.3/shapes/overflow_label_strategy.adoc

```diff
@@ -0,0 +1,61 @@
+= Overflow label strategy
+
+== Problem
+
+In the existing node layouts, the inside label of a node will always be used to determine the minimum size.
```

Comment:
What about outside label? Should we just exclude them? I'm not opposed to that, but we could have an outside label of thousands of words...

Reply:
For inside label, the maximum label width is defined by the node width, but for outside label we don't have such constraints.